### PR TITLE
HTS-1456: Decomission introduction page redirection flag

### PR DESCRIPTION
**Pull request by sureshhmrc**

### CODE REVIEW

**Comment by sureshhmrc** [comment]
REMOVED the introduction pages completely as we don't need them anymore or i should not do it ?

File: app/uk/gov/hmrc/helptosavefrontend/controllers/IntroductionController.scala [hunk 1]
```diff
@@ -41,48 +39,26 @@ class IntroductionController @Inject() (val authConnector: AuthConnector,
   extends BaseController with HelpToSaveAuth {
 
   def getAboutHelpToSave: Action[AnyContent] = unprotected { implicit request ⇒ implicit htsContext ⇒
-    handleIntroductionPage(
-      views.html.introduction.about_help_to_save(),
-      "https://www.gov.uk/get-help-savings-low-income"
-    )
+    SeeOther("https://www.gov.uk/get-help-savings-low-income")
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/helptosavefrontend/controllers/IntroductionController.scala]: it'd be more flexible for the future to read the beginning part of the URL from config (`govuk.urls.base`)

Comment (sureshhmrc) [reply app/uk/gov/hmrc/helptosavefrontend/controllers/IntroductionController.scala]: fixed

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**